README.md

```diff
@@ -5,6 +5,8 @@ And we're back 😤🎉! Use this repo to share and keep track of any tech-relat
 
 🤗 **Contribute by submitting a [pull request](https://github.com/susam/gitpr#create-pull-request) or [filling out this form](https://bit.ly/3d5O76c)!**  🤗
 
+[New Grad 2022 Listing](https://github.com/coderQuad/New-Grad-Positions-2022)
```

Comment:
Can we format this a bit better? Here's what we did [last year](https://github.com/pittcsc/Summer2022-Internships/tree/07d3871c0e021ca414ec8f2c94d3563bc06d386d)

Reply:
Fixed.